website/docs/docs/cloud/connect-data-platform/connect-onehouse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Onehouse"
+id: connect-onehouse
+description: "Setup instructions for connecting Onehouse to dbt Cloud"
+sidebar_label: "Connect Onehouse"
+pagination_next: null
+---
+
+<Snippet path="dbt-onehouse" />
+
+<Constant name="cloud" /> Supports connecting to [Onehouse SQL](https://www.onehouse.ai/) using the Apache Spark Connector with the Thrift method.
+
+**Connect to a Onehouse SQL Cluster with the `dbt-spark` adapter.**
+
+## Requirements
+
+* For dbt Cloud, ensure your Onehouse SQL endpoint is accessible via external DNS/IP, whitelisting dbt Cloud IPs.
+* For dbt Core, you can SSH tunnel into the VPC to access the endpoint, without opening up ports.
+
+## What works 
+
+* All dbt features like `dbt clean`, `dbt compile`, `dbt debug`, `dbt seed`, and `dbt run`.
+* dbt materialized type of "table" and "incremental" 
+* Apache Hudi table types of Merge on Read (MoR) and Copy on Write (CoW). It is recommended to use MoR for mutable workloads.
+
+## Limitations
+
+* dbt materialized type cannot be "view".
+* `dbt seed` has row / record limits.
```

Comment:
do you know the row/record limit?

Reply:
https://github.com/dbt-labs/dbt-adapters/issues/1040.  I put it at no more than 7500.

Reply:
Ah this issue :) Perhaps we should just call that out here?

Reply:
i'm hoping that dbt fill fix this.